packages/builder/cypress/integration/createScreen.js

```diff
@@ -7,5 +7,15 @@ context("Screen Tests", () => {
 
   it("Should successfully create a screen", () => {
     cy.createScreen("Test Screen", "/test")
+    cy.get(".nav-items-container").within(() => {
+      cy.contains("/test").should("exist")
+    })
+  })
+
+  it("Should update the url", () => {
+    cy.createScreen("Test Screen", "test with spaces")
+    cy.get(".nav-items-container").within(() => {
+      cy.contains("/test_with_spaces").should("exist")
```

Comment:
@mslourens looks like the cypress tests are failing due to the switch to dashes. Update this to:

```
test-with-spaces
```

and we are good to go.